docs/integrations/data-on-azure.md

```diff
@@ -38,7 +38,7 @@ This file should include the following information:
 
 You should then create a secret with this access keys information on Kubernetes on the same namespace as Polyaxon deployment:
 
-`kubectl create secret generic az-secret --from-file=az-secret.json=path/to/az-key.json -n polyaxon`
+`kubectl create secret generic az-secret --from-file=path/to/az-key.json -n polyaxon`
```

Comment:
This is a not a typo, that's the name the user can give to the secret.

Reply:
Figured that out, was a little confusion at first. :)

Reply:
No problem, the syntax in your PR is correct, however, in the docs it relies on the the name `az-secret.json`.